src/User/resources/views/admin/update.php

```diff
@@ -67,6 +67,10 @@
                                                 'label' => Yii::t('usuario', 'Assignments'),
                                                 'url' => ['/user/admin/assignments', 'id' => $user->id],
                                             ],
+                                            [
+                                                'label' => Yii::t('usuario', 'Session history'),
+                                                'url' => ['/user/admin/session-history', 'id' => $user->id],
```

Comment:
This should be visible only if `enableSessionHistory` is true

Reply:
Fix it.